src/Samples/AuthoringDemo/WinUI3CppApp/MainWindow.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<Window
+    x:Class="WinUI3CppApp.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinUI3CppApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
```

Comment:
Are we intentionally not using the custom types here until the WinAppSDK scenario is fixed?